lib/Tests/BluetoothEngineTests/GetDevicesTests.swift

```diff
@@ -0,0 +1,105 @@
+import Bluetooth
+@testable import BluetoothAction
+import BluetoothClient
+import BluetoothMessage
+import Foundation
+import JsMessage
+import Testing
+import XCTest
+
+extension Tag {
+    @Tag static var getDevices: Self
+}
+
+@Suite(.tags(.getDevices))
+struct GetDevicesResponseTests {
+    @Test
+    func toJsMessage_withZeroPeripherals_hasEmptyArrayBody() throws {
+        let sut = GetDevicesResponse(peripherals: [])
+        let jsMessage = sut.toJsMessage()
+        let body = try #require(jsMessage.extractBody(as: NSArray.self))
+        #expect(body == [])
+    }
+
+    @Test
+    func toJsMessage_withOnePeripheral_hasExpectedBody() throws {
+        let sut = GetDevicesResponse(peripherals: [(id: UUID(n: 0), name: "Slartibartfast")])
```

Comment:
42

Reply:
Yeah, Slartibartfast is a character from Hitchhiker's Guide to the Galaxy. In that series 42 is the answer to life's ultimate question.

Reply:
After millennia of computation Deep Thought provided that answer, which is why Slartibartfast's team had to proceed to build the Earth and make a backup copy.